terraform/modules/kafka/main.tf

```diff
@@ -23,6 +23,13 @@ resource "aws_msk_cluster" "kafka" {
     }
   }
 
+  encryption_info {
+    encryption_in_transit {
+      client_broker = var.client_broker_encryption_in_transit
```

Comment:
If we need this in all of the 3 projects I would suggest adding the values directly here as an indirection through variables which also have default values is not very readable and more complex